tests/test_travis_multiplePS/check_correctness_cirrus_sgd.cpp

```diff
@@ -0,0 +1,89 @@
+#include <unistd.h>
+#include <cstdlib>
```

Comment:
I don't know what this file is doing here. Nothing here seems to use multiple PSs

Reply:
This is a useless copy paste from the single PS test. I will remove this.